zulip/zulip/__init__.py

```diff
@@ -362,25 +370,25 @@ def __init__(self, email=None, api_key=None, config_file=None,
             self.tls_verification = False  # type: Union[bool, str]
         elif cert_bundle is not None:
             if not os.path.isfile(cert_bundle):
-                raise RuntimeError("tls bundle '%s' does not exist"
-                                   % (cert_bundle,))
+                raise FileNotFoundError("tls bundle '%s' does not exist"
+                                        % (cert_bundle,))
             self.tls_verification = cert_bundle
         else:
             # Default behavior: verify against system CA certificates
             self.tls_verification = True
 
         if client_cert is None:
             if client_cert_key is not None:
-                raise RuntimeError("client cert key '%s' specified, but no client cert public part provided"
-                                   % (client_cert_key,))
+                raise FileNotFoundError("client cert key '%s' specified, but no client cert public part provided"
```

Comment:
I'm sorry, I overlooked that client_cert and client_cert key can actually be specified as environment vars. Can you change it back to ZulipError for this and the following two exceptions?